marimo/_cli/export/commands.py

```diff
@@ -68,8 +69,9 @@ def write_data(data: str) -> None:
 
     if output:
         output_path = Path(output)
-        if not prompt_to_overwrite(output_path):
-            return
+        if output_path.exists() and not force:
```

Comment:
i think we can just do `if not force:`, because we check if the output exists in `prompt_to_overwrite`